### DISCUSSION TITLE
How to generate Dart code in snake_case format with flutter_rust_bridge?

### DISCUSSION THREAD

**Discussion by imxood** [fzyzcjy/flutter_rust_bridge#2309] [Q&A]
How to generate Dart code in snake_case format with flutter_rust_bridge?

**Comment by fzyzcjy** [comment]
Firstly, I wonder why do you want that? IIRC Dart code reuires camelCase and there may even be linters about this

**Reply by imxood** [reply]
When the Rust part generates a lot of Dart code, it becomes very inconvenient to find function calls on the Dart side due to the generated camelCase code when using snake-formatted functions from Rust. In Dart, I can ignore camel_case_types in the analysis_options.yaml file.

**Reply by fzyzcjy** [reply]
Hmm I see. Currently this is not supported yet and is not of top priority, since it seems to be good to stick to Dart style. However, feel free to PR to add this feature!
